Project.toml

```
@@ -10,7 +10,7 @@ SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 
 [compat]
 ExpectationStubs = "0.2"
-HTTP = "0.8.5"
+HTTP = "0.8, 0.9"
```

```suggestion
HTTP = "0.8.5, 0.9"
```

We ned at least 0.8.5, there is some feature that was added there for us (I think that I added)